rust/perspective-server/build/psp.rs

```diff
@@ -84,15 +84,15 @@ pub fn cmake_build() -> Result<Option<PathBuf>, std::io::Error> {
     // but our conda recipe doesn't directly invoke cmake
     if let Ok(cmake_args) = std::env::var("CMAKE_ARGS") {
         println!(
-            "cargo:warning=MESSAGE Setting CMAKE_ARGS from enviornment {:?}",
+            "cargo:warning=Setting CMAKE_ARGS from environment {:?}",
             cmake_args
         );
         for arg in Shlex::new(&cmake_args) {
             dst.configure_arg(arg);
         }
     }
 
-    println!("cargo:warning=MESSAGE Building cmake {}", profile);
+    println!("cargo:warning=Building cmake {}", profile);
```

Comment:
MESSAGE is not needed in these warnings, I think we misunderstood the docs. See this CI run: https://github.com/tomjakubowski/perspective-oneoff-tests/actions/runs/11752078774/job/32743108312